src/murmur/Cert.cpp

```diff
@@ -116,6 +117,27 @@ void Server::initializeCert() {
 		qlCA = ql;
 	}
 
+#if defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
+	if (! dhparams.isEmpty()) {
+		QSslDiffieHellmanParameters qdhp = QSslDiffieHellmanParameters(dhparams);
+		if (qdhp.isValid()) {
+			qsdhpDHParams = qdhp;
+		} else {
+			QSet<QSslDiffieHellmanParameters::Error> errors = qsdhpDHParams.errors();
```

Comment:
Updated the Qt changeset: https://codereview.qt-project.org/#/c/113070

...Updating win32-static buildenv and PR as we speak...